tests/app/clients/test_zendesk_sell.py

```diff
@@ -0,0 +1,53 @@
+import requests_mock
+
+from flask import Flask
+
+from app.clients.zendesk_sell_client import ZenDeskSellClient
+from app.user.contact_request import ContactRequest
+
+
+def test_create_lead(notify_api: Flask):
+    def match_json(request):
+        expected = {
+            'data': {
+                'last_name': 'User',
+                'first_name': 'Test',
+                'organization_name': '',
+                'email': 'test@email.com',
+                'description': 'Program: \n: ',
+                'tags': ["Support Request", "en"],
+                'status': 'New',
+                'custom_fields': {
+                    'Product': ['Notify'],
+                    'Source': 'Demo request form',
+                    'Intended recipients': 'No value'
```

Comment:
Cannot happen when called by the admin

Reply:
The tests within `test_zendesk_sell` and `test_freshdesk` combined cover both scenarios of `No value` and `internal` I think we are good - resolving conversation